.github/workflows/code-hygiene.yml

```diff
@@ -1,14 +1,13 @@
 name: Code Hygiene
 
-on: [pull_request]
-
+on: [push, pull_request, pull_request_target]
 jobs:
-  linelint:
+  spotless:
     runs-on: ubuntu-latest
-    name: Check if all files end in newline
+    name: Spotless Check
     steps:
       - name: Checkout
         uses: actions/checkout@v2
 
-      - name: Linelint
-        uses: fernandrone/linelint@0.0.4
+      - name: Spotless Check
+        run: ./gradlew spotlessCheck
```

Comment:
Spotless is run with precommit. We can remove this file

Reply:
We don’t have a precommit workflow on SDK.

Reply:
Ah. I meant gradle check. Also, get confused with sdk and OpenSearch repo. My bad

Reply:
The point of this setup is to do a “fail fast” on spotless before calling gradle check which takes several minutes before getting that far.

Reply:
Makes sense!